app/backend/src/couchers/email/__init__.py

```diff
@@ -12,16 +12,25 @@
 loader = FileSystemLoader(Path(__file__).parent / ".." / ".." / ".." / "templates")
 env = Environment(loader=loader, trim_blocks=True)
 
-plain_base_template = env.get_template("email_base_plain.md")
-html_base_template = env.get_template("email_base_html.html")
+
+def couchers_escape(value):
+    return escape(str(value))
+
+
+def couchers_safe(value):
+    return value
 
 
-def _escape_plain(text):
-    return text
+env.filters["couchers_escape"] = couchers_escape
+env.filters["couchers_safe"] = couchers_safe
+
+plain_base_template = env.get_template("email_base_plain.md")
+html_base_template = env.get_template("email_base_html.html")
 
 
-def _escape_html(text):
-    return text.replace("_", "\\_")
+def render_html(text):
+    stripped_paragraphs = text.strip().split("\n\n")
```

Comment:
Not immediately obvious to me why we need this, maybe a short doc string or comment here?